src/frontend/src/helpers/Storage/Interface.ts

```diff
@@ -0,0 +1,80 @@
+export {AbstractStorage,
+        File, FileID, FileBrief,
+        Project, ProjectID, ProjectBrief,
+        Settings, defaultSettings}
+
+abstract class AbstractStorage {
```

Comment:
In the future, the FileID won't hold the project information, so it would be good if we went back to the original contracts and passed along the ProjectID.